### PR TITLE
Feat/cip 95 dapp e2e tests

### PR DESCRIPTION
**Pull request by mirceahasegan**
# Context
LW-7977
LW-8227
SDK e2e tests for cip-95 dapp integration.
For now, only the initialization of the test with the enable function is done.
Once CIP-95 functionality is merged, tests will be added in this new test file.

# Proposed Solution
1. Refactor existing web-extension tests to separate test files for: dapp-cip30 and web-extension.
2. Add built cip-95 dapp and a test file that initialises it just as we do for cip-30

# TODO
- [x] tests for getActiveDrepId and getActivePublicStakekeys

### CODE REVIEW

**Review by mkazlauskas — APPROVED** [review]
Nice refactor :muscle:

**Review by mchappell — CHANGES_REQUESTED** [review]

File: packages/e2e/test/web-extension/dapp-cip95/.DS_Store
[the remark this answers is not in the document]
Comment (mirceahasegan) [reply packages/e2e/test/web-extension/dapp-cip95/.DS_Store:1]: I'll remove them at the final squash. Let's leave this comment open as a reminder 😄

[the remark this answers is not in the document]
Comment (mirceahasegan) [reply packages/e2e/test/web-extension/dapp-cip95/.DS_Store:1]: Removed here: [5e37c21](https://github.com/input-output-hk/cardano-js-sdk/pull/891/commits/5e37c216c93de6a74620ceee41647f44704469a8).
Marked them to be ignored in the future: [f30edbd](https://github.com/input-output-hk/cardano-js-sdk/pull/891/commits/f30edbd5d54dcf71080c8527f8437a7a68bed03b)

**Review by lgobbi-atix — APPROVED** [review]
LGTM

File: packages/e2e/test/web-extension/dapp-cip95/.DS_Store
[the remark this answers is not in the document]
Comment (mkazlauskas) [reply packages/e2e/test/web-extension/dapp-cip95/.DS_Store:1]: Please squash the removal commit to 7d37ebd324d69a70219699c7150a4a861deb2dfa

**Review by mkazlauskas — APPROVED** [review]
Great work! :rocket:

**Review by mchappell — APPROVED** [review]
Great work, nice refactoring to make the selectors easier to access between multiple DApps 🥳

**Pull request merged**